internal/topo/transform/template.go

```diff
@@ -53,7 +53,7 @@
 		}
 		tp = temp
 	}
-	return func(d interface{}) ([]byte, bool, error) {
+	return func(d interface{}, s bool) ([]byte, bool, error) {
```

Comment:
Add some comments about the meaning of s.
The flow in this function is still a bit messy. If s=false means skip fields/format, we can check it only after tp and before format?